fixtures/eurolite/led-fe-1500.json

```diff
@@ -6,11 +6,14 @@
   "meta": {
     "authors": ["Felix Edelmann"],
     "createDate": "2017-12-31",
-    "lastModifyDate": "2018-08-09"
+    "lastModifyDate": "2024-11-23"
   },
   "links": {
+    "productPage": [
```

Comment:
Can you please put `productPage` after `manual` for consistency (in all the places you added it)?

Reply:
Fixed link order. I noticed this pattern after I had already sent the pull request.

Reply:
No worries :slightly_smiling_face: